pages/article/[id].js

```diff
@@ -10,22 +13,99 @@ import useCurrentUser from 'lib/useCurrentUser';
 import { nl2br, linkify, ellipsis } from 'lib/text';
 import { usePushToDataLayer } from 'lib/gtm';
 
+import { format, formatDistanceToNow } from 'lib/dateWithLocale';
+import isValid from 'date-fns/isValid';
+
 import AppLayout from 'components/AppLayout';
 import Hyperlinks from 'components/Hyperlinks';
-import ArticleInfo from 'components/ArticleInfo';
-import Trendline from 'components/Trendline';
 import CurrentReplies from 'components/CurrentReplies';
 import ReplyRequestReason from 'components/ReplyRequestReason';
-import CreateReplyRequestDialog from 'components/CreateReplyRequestDialog';
+import CreateReplyRequestForm from 'components/CreateReplyRequestForm';
 import NewReplySection from 'components/NewReplySection';
 import ArticleItem from 'components/ArticleItem';
+import ArticleInfo from 'components/ArticleInfo';
 import ArticleCategories from 'components/ArticleCategories';
+import cx from 'clsx';
+import Trendline from 'components/Trendline';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    display: 'flex',
+    padding: '24px 0',
+    flexDirection: 'column',
+    [theme.breakpoints.up('md')]: {
+      flexDirection: 'row',
+      alignItems: 'baseline',
+    },
+  },
+  card: {
+    background: theme.palette.common.white,
+    borderRadius: 8,
+  },
+  main: {
+    flex: 1,
+    marginRight: 0,
+    [theme.breakpoints.up('md')]: {
+      flex: 3,
+      marginRight: 12,
+    },
+  },
+  aside: {
+    background: 'transparent',
+    [theme.breakpoints.up('md')]: {
+      padding: '16px 28px',
+      background: theme.palette.common.white,
```

Comment:
seems that we are missing `flex: 1` here
![image](https://user-images.githubusercontent.com/108608/82549705-ebfbd580-9b8f-11ea-8a54-c14b53537f7f.png)